mirdata/datasets/mtg_jamendo_autotagging_moodtheme.py

```diff
@@ -0,0 +1,340 @@
+# -*- coding: utf-8 -*-
+"""MTG_jamendo_autotagging_moodtheme Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    The MTG Jamendo autotagging mood/theme Dataset is a new open dataset for music auto-tagging. It
```

Comment:
Can we include the website somewhere in this description? https://github.com/MTG/mtg-jamendo-dataset

Reply:
done